Tests disabled due to possible JVM crash are generated

**Description**

Tests disabled due to possible JVM crash are generated.
This is actual for simple code samples with regular test generation (without Spring support).

**To Reproduce**

1. Install [UnitTestBot plugin built from main](https://github.com/UnitTestBot/UTBotJava/actions/runs/5223069802) in IntelliJ IDEA Ultimate 2023.1
2. Open/clone UTBotJava project
3. Generate tests for a sample - f.i. `algorithms/Sort.java` or `stream/BaseStreamExample.java` with default settings

**Expected behavior**

Simple samples are not expected to crash JVM.
Generated tests are expected to be not disabled.
Tests are expected to save result of a method call and assert with expected.

**Actual behavior**

All generated tests are annotated with `@org.junit.jupiter.api.Disabled(value = "Disabled due to possible JVM crash")`
and do not have any result assertions.

**Screenshots, logs**

![image](https://github.com/UnitTestBot/UTBotJava/assets/37301492/1c188e53-1045-4bf9-9311-c2ffea7cff7a)

~~~java
public final class SortTest {
    ///region Test suites for executable org.utbot.examples.algorithms.Sort.quickSort


    @Test
    @org.junit.jupiter.api.Disabled(value = "Disabled due to possible JVM crash")
    public void testQuickSort1() {
        Sort sort = new Sort();

        // This invocation possibly crashes JVM
        sort.quickSort(null, 96, 96);
    }

    ///endregion

    ///region Test suites for executable org.utbot.examples.algorithms.Sort.swap


    @Test
    @org.junit.jupiter.api.Disabled(value = "Disabled due to possible JVM crash")
    public void testSwap1() {
        Sort sort = new Sort();

        // This invocation possibly crashes JVM
        sort.swap(null, -255, -255);
    }

    ///endregion
}
~~~

<details>
<summary>
There are many similar errors in utbot-engine-current.log
</summary>
20:22:21.167 | ERROR | ConcreteExecutor          | executeAsync, response(ERROR)
org.utbot.framework.plugin.api.InstrumentedProcessDeathException: 
----------------------------------------
The instrumented process is dead
Cause:
Could not initialize class org.utbot.instrumentation.rd.InstrumentedProcessKt
----------------------------------------

	at org.utbot.instrumentation.ConcreteExecutor.withProcess$throwConcreteIfDead(ConcreteExecutor.kt:183) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.ConcreteExecutor.withProcess(ConcreteExecutor.kt:213) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.ConcreteExecutor.withProcess$default(ConcreteExecutor.kt:180) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:228) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.engine.UtBotSymbolicEngineKt.executeConcretely(UtBotSymbolicEngine.kt:687) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.engine.UtBotSymbolicEngineKt.access$executeConcretely(UtBotSymbolicEngine.kt:1) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.engine.UtBotSymbolicEngine$fuzzing$2$1.invokeSuspend(UtBotSymbolicEngine.kt:429) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.engine.UtBotSymbolicEngine$fuzzing$2$1.invoke(UtBotSymbolicEngine.kt) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.engine.UtBotSymbolicEngine$fuzzing$2$1.invoke(UtBotSymbolicEngine.kt) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.JavaLanguageKt$runJavaFuzzing$5.invokeSuspend(JavaLanguage.kt:129) ~[utbot-java-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.JavaLanguageKt$runJavaFuzzing$5.invoke(JavaLanguage.kt) ~[utbot-java-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.JavaLanguageKt$runJavaFuzzing$5.invoke(JavaLanguage.kt) ~[utbot-java-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.BaseFuzzing.handle(Providers.kt:56) ~[utbot-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.FuzzingApi.fuzz(Api.kt:350) ~[utbot-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.FuzzingApi.access$fuzz(Api.kt:1) ~[utbot-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.FuzzingApi$fuzz$2.invokeSuspend(Api.kt) ~[utbot-fuzzing-main-2023.6.4562.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) ~[kotlin-stdlib-1.8.10.jar:1.8.10-release-430(1.8.10)]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:284) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at org.utbot.common.ConcurrencyKt.runBlockingWithCancellationPredicate(Concurrency.kt:38) ~[utbot-core-main-2023.6.4562.jar:?]
	at org.utbot.framework.plugin.api.TestCaseGenerator$generate$3.invoke(TestCaseGenerator.kt:174) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.framework.plugin.api.TestCaseGenerator$generate$3.invoke(TestCaseGenerator.kt:173) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.common.ConcurrencyKt.runIgnoringCancellationException(Concurrency.kt:47) ~[utbot-core-main-2023.6.4562.jar:?]
	at org.utbot.framework.plugin.api.TestCaseGenerator.generate(TestCaseGenerator.kt:173) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.framework.process.EngineProcessMainKt$setup$4.invoke(EngineProcessMain.kt:135) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.framework.process.EngineProcessMainKt$setup$4.invoke(EngineProcessMain.kt:120) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.rd.IdleWatchdog$measureTimeForActiveCall$1$2$1.invoke(ClientProcessUtil.kt:115) ~[utbot-rd-main-2023.6.4562.jar:?]
	at org.utbot.rd.IdleWatchdog.wrapActive(ClientProcessUtil.kt:88) ~[utbot-rd-main-2023.6.4562.jar:?]
	at org.utbot.rd.IdleWatchdog$measureTimeForActiveCall$1.invoke(ClientProcessUtil.kt:114) ~[utbot-rd-main-2023.6.4562.jar:?]
	at com.jetbrains.rd.framework.IRdEndpoint$set$1.invoke(TaskInterfaces.kt:182) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.IRdEndpoint$set$1.invoke(TaskInterfaces.kt:182) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.impl.RdCall.onWireReceived(RdTask.kt:362) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.MessageBroker$invoke$2$2.invoke(MessageBroker.kt:57) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.MessageBroker$invoke$2$2.invoke(MessageBroker.kt:56) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.impl.ProtocolContexts.readMessageContextAndInvoke(ProtocolContexts.kt:148) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.MessageBroker$invoke$2.invoke(MessageBroker.kt:56) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.MessageBroker$invoke$2.invoke(MessageBroker.kt:54) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.util.threading.SingleThreadSchedulerBase.queue$lambda-3(SingleThreadScheduler.kt:41) ~[rd-core-2023.1.2.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.NoClassDefFoundError: Could not initialize class org.utbot.instrumentation.rd.InstrumentedProcessKt
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion$1.invoke(InstrumentedProcess.kt:103) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion$1.invoke(InstrumentedProcess.kt:101) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.framework.process.AbstractRDProcessCompanion.obtainProcessCommandLine(AbstractRDProcessCompanion.kt:20) ~[utbot-framework-api-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion.access$obtainProcessCommandLine(InstrumentedProcess.kt:97) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion$invoke$2$rdProcess$1.invoke(InstrumentedProcess.kt:121) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion$invoke$2$rdProcess$1.invoke(InstrumentedProcess.kt:118) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.rd.UtRdUtilKt.startUtProcessWithRdServer(UtRdUtil.kt:115) ~[utbot-rd-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion.invoke(InstrumentedProcess.kt:118) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.ConcreteExecutor.regenerate(ConcreteExecutor.kt:160) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.ConcreteExecutor.withProcess(ConcreteExecutor.kt:199) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	... 44 more
20:22:21.169 | ERROR | ConcreteExecutor          | executeAsync, response(ERROR)
org.utbot.framework.plugin.api.InstrumentedProcessDeathException: 
----------------------------------------
The instrumented process is dead
Cause:
Could not initialize class org.utbot.instrumentation.rd.InstrumentedProcessKt
----------------------------------------

	at org.utbot.instrumentation.ConcreteExecutor.withProcess$throwConcreteIfDead(ConcreteExecutor.kt:183) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.ConcreteExecutor.withProcess(ConcreteExecutor.kt:213) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.ConcreteExecutor.withProcess$default(ConcreteExecutor.kt:180) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:228) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.engine.UtBotSymbolicEngineKt.executeConcretely(UtBotSymbolicEngine.kt:687) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.engine.UtBotSymbolicEngineKt.access$executeConcretely(UtBotSymbolicEngine.kt:1) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.engine.UtBotSymbolicEngine$fuzzing$2$1.invokeSuspend(UtBotSymbolicEngine.kt:429) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.engine.UtBotSymbolicEngine$fuzzing$2$1.invoke(UtBotSymbolicEngine.kt) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.engine.UtBotSymbolicEngine$fuzzing$2$1.invoke(UtBotSymbolicEngine.kt) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.JavaLanguageKt$runJavaFuzzing$5.invokeSuspend(JavaLanguage.kt:129) ~[utbot-java-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.JavaLanguageKt$runJavaFuzzing$5.invoke(JavaLanguage.kt) ~[utbot-java-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.JavaLanguageKt$runJavaFuzzing$5.invoke(JavaLanguage.kt) ~[utbot-java-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.BaseFuzzing.handle(Providers.kt:56) ~[utbot-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.FuzzingApi.fuzz(Api.kt:350) ~[utbot-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.FuzzingApi.access$fuzz(Api.kt:1) ~[utbot-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.FuzzingApi$fuzz$2.invokeSuspend(Api.kt) ~[utbot-fuzzing-main-2023.6.4562.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) ~[kotlin-stdlib-1.8.10.jar:1.8.10-release-430(1.8.10)]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:284) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at org.utbot.common.ConcurrencyKt.runBlockingWithCancellationPredicate(Concurrency.kt:38) ~[utbot-core-main-2023.6.4562.jar:?]
	at org.utbot.framework.plugin.api.TestCaseGenerator$generate$3.invoke(TestCaseGenerator.kt:174) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.framework.plugin.api.TestCaseGenerator$generate$3.invoke(TestCaseGenerator.kt:173) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.common.ConcurrencyKt.runIgnoringCancellationException(Concurrency.kt:47) ~[utbot-core-main-2023.6.4562.jar:?]
	at org.utbot.framework.plugin.api.TestCaseGenerator.generate(TestCaseGenerator.kt:173) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.framework.process.EngineProcessMainKt$setup$4.invoke(EngineProcessMain.kt:135) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.framework.process.EngineProcessMainKt$setup$4.invoke(EngineProcessMain.kt:120) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.rd.IdleWatchdog$measureTimeForActiveCall$1$2$1.invoke(ClientProcessUtil.kt:115) ~[utbot-rd-main-2023.6.4562.jar:?]
	at org.utbot.rd.IdleWatchdog.wrapActive(ClientProcessUtil.kt:88) ~[utbot-rd-main-2023.6.4562.jar:?]
	at org.utbot.rd.IdleWatchdog$measureTimeForActiveCall$1.invoke(ClientProcessUtil.kt:114) ~[utbot-rd-main-2023.6.4562.jar:?]
	at com.jetbrains.rd.framework.IRdEndpoint$set$1.invoke(TaskInterfaces.kt:182) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.IRdEndpoint$set$1.invoke(TaskInterfaces.kt:182) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.impl.RdCall.onWireReceived(RdTask.kt:362) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.MessageBroker$invoke$2$2.invoke(MessageBroker.kt:57) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.MessageBroker$invoke$2$2.invoke(MessageBroker.kt:56) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.impl.ProtocolContexts.readMessageContextAndInvoke(ProtocolContexts.kt:148) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.MessageBroker$invoke$2.invoke(MessageBroker.kt:56) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.MessageBroker$invoke$2.invoke(MessageBroker.kt:54) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.util.threading.SingleThreadSchedulerBase.queue$lambda-3(SingleThreadScheduler.kt:41) ~[rd-core-2023.1.2.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.NoClassDefFoundError: Could not initialize class org.utbot.instrumentation.rd.InstrumentedProcessKt
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion$1.invoke(InstrumentedProcess.kt:103) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion$1.invoke(InstrumentedProcess.kt:101) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.framework.process.AbstractRDProcessCompanion.obtainProcessCommandLine(AbstractRDProcessCompanion.kt:20) ~[utbot-framework-api-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion.access$obtainProcessCommandLine(InstrumentedProcess.kt:97) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion$invoke$2$rdProcess$1.invoke(InstrumentedProcess.kt:121) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion$invoke$2$rdProcess$1.invoke(InstrumentedProcess.kt:118) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.rd.UtRdUtilKt.startUtProcessWithRdServer(UtRdUtil.kt:115) ~[utbot-rd-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion.invoke(InstrumentedProcess.kt:118) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.ConcreteExecutor.regenerate(ConcreteExecutor.kt:160) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.ConcreteExecutor.withProcess(ConcreteExecutor.kt:199) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	... 44 more
20:22:21.169 | ERROR | ConcreteExecutor          | executeAsync, response(ERROR)
org.utbot.framework.plugin.api.InstrumentedProcessDeathException: 
----------------------------------------
The instrumented process is dead
Cause:
Could not initialize class org.utbot.instrumentation.rd.InstrumentedProcessKt
----------------------------------------

	at org.utbot.instrumentation.ConcreteExecutor.withProcess$throwConcreteIfDead(ConcreteExecutor.kt:183) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.ConcreteExecutor.withProcess(ConcreteExecutor.kt:213) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.ConcreteExecutor.withProcess$default(ConcreteExecutor.kt:180) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:228) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.engine.UtBotSymbolicEngineKt.executeConcretely(UtBotSymbolicEngine.kt:687) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.engine.UtBotSymbolicEngineKt.access$executeConcretely(UtBotSymbolicEngine.kt:1) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.engine.UtBotSymbolicEngine$fuzzing$2$1.invokeSuspend(UtBotSymbolicEngine.kt:429) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.engine.UtBotSymbolicEngine$fuzzing$2$1.invoke(UtBotSymbolicEngine.kt) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.engine.UtBotSymbolicEngine$fuzzing$2$1.invoke(UtBotSymbolicEngine.kt) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.JavaLanguageKt$runJavaFuzzing$5.invokeSuspend(JavaLanguage.kt:129) ~[utbot-java-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.JavaLanguageKt$runJavaFuzzing$5.invoke(JavaLanguage.kt) ~[utbot-java-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.JavaLanguageKt$runJavaFuzzing$5.invoke(JavaLanguage.kt) ~[utbot-java-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.BaseFuzzing.handle(Providers.kt:56) ~[utbot-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.FuzzingApi.fuzz(Api.kt:350) ~[utbot-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.FuzzingApi.access$fuzz(Api.kt:1) ~[utbot-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.FuzzingApi$fuzz$2.invokeSuspend(Api.kt) ~[utbot-fuzzing-main-2023.6.4562.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) ~[kotlin-stdlib-1.8.10.jar:1.8.10-release-430(1.8.10)]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:284) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at org.utbot.common.ConcurrencyKt.runBlockingWithCancellationPredicate(Concurrency.kt:38) ~[utbot-core-main-2023.6.4562.jar:?]
	at org.utbot.framework.plugin.api.TestCaseGenerator$generate$3.invoke(TestCaseGenerator.kt:174) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.framework.plugin.api.TestCaseGenerator$generate$3.invoke(TestCaseGenerator.kt:173) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.common.ConcurrencyKt.runIgnoringCancellationException(Concurrency.kt:47) ~[utbot-core-main-2023.6.4562.jar:?]
	at org.utbot.framework.plugin.api.TestCaseGenerator.generate(TestCaseGenerator.kt:173) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.framework.process.EngineProcessMainKt$setup$4.invoke(EngineProcessMain.kt:135) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.framework.process.EngineProcessMainKt$setup$4.invoke(EngineProcessMain.kt:120) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.rd.IdleWatchdog$measureTimeForActiveCall$1$2$1.invoke(ClientProcessUtil.kt:115) ~[utbot-rd-main-2023.6.4562.jar:?]
	at org.utbot.rd.IdleWatchdog.wrapActive(ClientProcessUtil.kt:88) ~[utbot-rd-main-2023.6.4562.jar:?]
	at org.utbot.rd.IdleWatchdog$measureTimeForActiveCall$1.invoke(ClientProcessUtil.kt:114) ~[utbot-rd-main-2023.6.4562.jar:?]
	at com.jetbrains.rd.framework.IRdEndpoint$set$1.invoke(TaskInterfaces.kt:182) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.IRdEndpoint$set$1.invoke(TaskInterfaces.kt:182) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.impl.RdCall.onWireReceived(RdTask.kt:362) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.MessageBroker$invoke$2$2.invoke(MessageBroker.kt:57) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.MessageBroker$invoke$2$2.invoke(MessageBroker.kt:56) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.impl.ProtocolContexts.readMessageContextAndInvoke(ProtocolContexts.kt:148) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.MessageBroker$invoke$2.invoke(MessageBroker.kt:56) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.MessageBroker$invoke$2.invoke(MessageBroker.kt:54) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.util.threading.SingleThreadSchedulerBase.queue$lambda-3(SingleThreadScheduler.kt:41) ~[rd-core-2023.1.2.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.NoClassDefFoundError: Could not initialize class org.utbot.instrumentation.rd.InstrumentedProcessKt
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion$1.invoke(InstrumentedProcess.kt:103) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion$1.invoke(InstrumentedProcess.kt:101) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.framework.process.AbstractRDProcessCompanion.obtainProcessCommandLine(AbstractRDProcessCompanion.kt:20) ~[utbot-framework-api-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion.access$obtainProcessCommandLine(InstrumentedProcess.kt:97) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion$invoke$2$rdProcess$1.invoke(InstrumentedProcess.kt:121) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion$invoke$2$rdProcess$1.invoke(InstrumentedProcess.kt:118) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.rd.UtRdUtilKt.startUtProcessWithRdServer(UtRdUtil.kt:115) ~[utbot-rd-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion.invoke(InstrumentedProcess.kt:118) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.ConcreteExecutor.regenerate(ConcreteExecutor.kt:160) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.ConcreteExecutor.withProcess(ConcreteExecutor.kt:199) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	... 44 more
20:22:21.171 | ERROR | ConcreteExecutor          | executeAsync, response(ERROR)
org.utbot.framework.plugin.api.InstrumentedProcessDeathException: 
----------------------------------------
The instrumented process is dead
Cause:
Could not initialize class org.utbot.instrumentation.rd.InstrumentedProcessKt
----------------------------------------

	at org.utbot.instrumentation.ConcreteExecutor.withProcess$throwConcreteIfDead(ConcreteExecutor.kt:183) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.ConcreteExecutor.withProcess(ConcreteExecutor.kt:213) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.ConcreteExecutor.withProcess$default(ConcreteExecutor.kt:180) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:228) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.engine.UtBotSymbolicEngineKt.executeConcretely(UtBotSymbolicEngine.kt:687) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.engine.UtBotSymbolicEngineKt.access$executeConcretely(UtBotSymbolicEngine.kt:1) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.engine.UtBotSymbolicEngine$fuzzing$2$1.invokeSuspend(UtBotSymbolicEngine.kt:429) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.engine.UtBotSymbolicEngine$fuzzing$2$1.invoke(UtBotSymbolicEngine.kt) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.engine.UtBotSymbolicEngine$fuzzing$2$1.invoke(UtBotSymbolicEngine.kt) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.JavaLanguageKt$runJavaFuzzing$5.invokeSuspend(JavaLanguage.kt:129) ~[utbot-java-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.JavaLanguageKt$runJavaFuzzing$5.invoke(JavaLanguage.kt) ~[utbot-java-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.JavaLanguageKt$runJavaFuzzing$5.invoke(JavaLanguage.kt) ~[utbot-java-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.BaseFuzzing.handle(Providers.kt:56) ~[utbot-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.FuzzingApi.fuzz(Api.kt:350) ~[utbot-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.FuzzingApi.access$fuzz(Api.kt:1) ~[utbot-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.FuzzingApi$fuzz$2.invokeSuspend(Api.kt) ~[utbot-fuzzing-main-2023.6.4562.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) ~[kotlin-stdlib-1.8.10.jar:1.8.10-release-430(1.8.10)]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:284) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at org.utbot.common.ConcurrencyKt.runBlockingWithCancellationPredicate(Concurrency.kt:38) ~[utbot-core-main-2023.6.4562.jar:?]
	at org.utbot.framework.plugin.api.TestCaseGenerator$generate$3.invoke(TestCaseGenerator.kt:174) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.framework.plugin.api.TestCaseGenerator$generate$3.invoke(TestCaseGenerator.kt:173) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.common.ConcurrencyKt.runIgnoringCancellationException(Concurrency.kt:47) ~[utbot-core-main-2023.6.4562.jar:?]
	at org.utbot.framework.plugin.api.TestCaseGenerator.generate(TestCaseGenerator.kt:173) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.framework.process.EngineProcessMainKt$setup$4.invoke(EngineProcessMain.kt:135) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.framework.process.EngineProcessMainKt$setup$4.invoke(EngineProcessMain.kt:120) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.rd.IdleWatchdog$measureTimeForActiveCall$1$2$1.invoke(ClientProcessUtil.kt:115) ~[utbot-rd-main-2023.6.4562.jar:?]
	at org.utbot.rd.IdleWatchdog.wrapActive(ClientProcessUtil.kt:88) ~[utbot-rd-main-2023.6.4562.jar:?]
	at org.utbot.rd.IdleWatchdog$measureTimeForActiveCall$1.invoke(ClientProcessUtil.kt:114) ~[utbot-rd-main-2023.6.4562.jar:?]
	at com.jetbrains.rd.framework.IRdEndpoint$set$1.invoke(TaskInterfaces.kt:182) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.IRdEndpoint$set$1.invoke(TaskInterfaces.kt:182) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.impl.RdCall.onWireReceived(RdTask.kt:362) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.MessageBroker$invoke$2$2.invoke(MessageBroker.kt:57) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.MessageBroker$invoke$2$2.invoke(MessageBroker.kt:56) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.impl.ProtocolContexts.readMessageContextAndInvoke(ProtocolContexts.kt:148) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.MessageBroker$invoke$2.invoke(MessageBroker.kt:56) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.MessageBroker$invoke$2.invoke(MessageBroker.kt:54) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.util.threading.SingleThreadSchedulerBase.queue$lambda-3(SingleThreadScheduler.kt:41) ~[rd-core-2023.1.2.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.NoClassDefFoundError: Could not initialize class org.utbot.instrumentation.rd.InstrumentedProcessKt
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion$1.invoke(InstrumentedProcess.kt:103) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion$1.invoke(InstrumentedProcess.kt:101) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.framework.process.AbstractRDProcessCompanion.obtainProcessCommandLine(AbstractRDProcessCompanion.kt:20) ~[utbot-framework-api-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion.access$obtainProcessCommandLine(InstrumentedProcess.kt:97) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion$invoke$2$rdProcess$1.invoke(InstrumentedProcess.kt:121) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion$invoke$2$rdProcess$1.invoke(InstrumentedProcess.kt:118) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.rd.UtRdUtilKt.startUtProcessWithRdServer(UtRdUtil.kt:115) ~[utbot-rd-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion.invoke(InstrumentedProcess.kt:118) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.ConcreteExecutor.regenerate(ConcreteExecutor.kt:160) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.ConcreteExecutor.withProcess(ConcreteExecutor.kt:199) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	... 44 more
20:22:21.173 | ERROR | ConcreteExecutor          | executeAsync, response(ERROR)
org.utbot.framework.plugin.api.InstrumentedProcessDeathException: 
----------------------------------------
The instrumented process is dead
Cause:
Could not initialize class org.utbot.instrumentation.rd.InstrumentedProcessKt
----------------------------------------

	at org.utbot.instrumentation.ConcreteExecutor.withProcess$throwConcreteIfDead(ConcreteExecutor.kt:183) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.ConcreteExecutor.withProcess(ConcreteExecutor.kt:213) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.ConcreteExecutor.withProcess$default(ConcreteExecutor.kt:180) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:228) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.engine.UtBotSymbolicEngineKt.executeConcretely(UtBotSymbolicEngine.kt:687) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.engine.UtBotSymbolicEngineKt.access$executeConcretely(UtBotSymbolicEngine.kt:1) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.engine.UtBotSymbolicEngine$fuzzing$2$1.invokeSuspend(UtBotSymbolicEngine.kt:429) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.engine.UtBotSymbolicEngine$fuzzing$2$1.invoke(UtBotSymbolicEngine.kt) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.engine.UtBotSymbolicEngine$fuzzing$2$1.invoke(UtBotSymbolicEngine.kt) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.JavaLanguageKt$runJavaFuzzing$5.invokeSuspend(JavaLanguage.kt:129) ~[utbot-java-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.JavaLanguageKt$runJavaFuzzing$5.invoke(JavaLanguage.kt) ~[utbot-java-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.JavaLanguageKt$runJavaFuzzing$5.invoke(JavaLanguage.kt) ~[utbot-java-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.BaseFuzzing.handle(Providers.kt:56) ~[utbot-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.FuzzingApi.fuzz(Api.kt:350) ~[utbot-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.FuzzingApi.access$fuzz(Api.kt:1) ~[utbot-fuzzing-main-2023.6.4562.jar:?]
	at org.utbot.fuzzing.FuzzingApi$fuzz$2.invokeSuspend(Api.kt) ~[utbot-fuzzing-main-2023.6.4562.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) ~[kotlin-stdlib-1.8.10.jar:1.8.10-release-430(1.8.10)]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:284) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source) ~[kotlinx-coroutines-core-jvm-1.6.3.jar:?]
	at org.utbot.common.ConcurrencyKt.runBlockingWithCancellationPredicate(Concurrency.kt:38) ~[utbot-core-main-2023.6.4562.jar:?]
	at org.utbot.framework.plugin.api.TestCaseGenerator$generate$3.invoke(TestCaseGenerator.kt:174) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.framework.plugin.api.TestCaseGenerator$generate$3.invoke(TestCaseGenerator.kt:173) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.common.ConcurrencyKt.runIgnoringCancellationException(Concurrency.kt:47) ~[utbot-core-main-2023.6.4562.jar:?]
	at org.utbot.framework.plugin.api.TestCaseGenerator.generate(TestCaseGenerator.kt:173) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.framework.process.EngineProcessMainKt$setup$4.invoke(EngineProcessMain.kt:135) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.framework.process.EngineProcessMainKt$setup$4.invoke(EngineProcessMain.kt:120) ~[utbot-framework-main-2023.6.4562.jar:?]
	at org.utbot.rd.IdleWatchdog$measureTimeForActiveCall$1$2$1.invoke(ClientProcessUtil.kt:115) ~[utbot-rd-main-2023.6.4562.jar:?]
	at org.utbot.rd.IdleWatchdog.wrapActive(ClientProcessUtil.kt:88) ~[utbot-rd-main-2023.6.4562.jar:?]
	at org.utbot.rd.IdleWatchdog$measureTimeForActiveCall$1.invoke(ClientProcessUtil.kt:114) ~[utbot-rd-main-2023.6.4562.jar:?]
	at com.jetbrains.rd.framework.IRdEndpoint$set$1.invoke(TaskInterfaces.kt:182) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.IRdEndpoint$set$1.invoke(TaskInterfaces.kt:182) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.impl.RdCall.onWireReceived(RdTask.kt:362) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.MessageBroker$invoke$2$2.invoke(MessageBroker.kt:57) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.MessageBroker$invoke$2$2.invoke(MessageBroker.kt:56) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.impl.ProtocolContexts.readMessageContextAndInvoke(ProtocolContexts.kt:148) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.MessageBroker$invoke$2.invoke(MessageBroker.kt:56) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.framework.MessageBroker$invoke$2.invoke(MessageBroker.kt:54) ~[rd-framework-2023.1.2.jar:?]
	at com.jetbrains.rd.util.threading.SingleThreadSchedulerBase.queue$lambda-3(SingleThreadScheduler.kt:41) ~[rd-core-2023.1.2.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.NoClassDefFoundError: Could not initialize class org.utbot.instrumentation.rd.InstrumentedProcessKt
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion$1.invoke(InstrumentedProcess.kt:103) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion$1.invoke(InstrumentedProcess.kt:101) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.framework.process.AbstractRDProcessCompanion.obtainProcessCommandLine(AbstractRDProcessCompanion.kt:20) ~[utbot-framework-api-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion.access$obtainProcessCommandLine(InstrumentedProcess.kt:97) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion$invoke$2$rdProcess$1.invoke(InstrumentedProcess.kt:121) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion$invoke$2$rdProcess$1.invoke(InstrumentedProcess.kt:118) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.rd.UtRdUtilKt.startUtProcessWithRdServer(UtRdUtil.kt:115) ~[utbot-rd-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.rd.InstrumentedProcess$Companion.invoke(InstrumentedProcess.kt:118) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.ConcreteExecutor.regenerate(ConcreteExecutor.kt:160) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	at org.utbot.instrumentation.ConcreteExecutor.withProcess(ConcreteExecutor.kt:199) ~[utbot-instrumentation-main-2023.6.4562.jar:?]
	... 44 more
20:22:21.174 | ERROR | ConcreteExecutor          | executeAsync, response(ERROR)
org.utbot.framework.plugin.api.InstrumentedProcessDeathException: 
</details>

**Environment**

IntelliJ IDEA version - 2023.1 Ultimate
Project - Gradle
JDK - 17

--- a/utbot-instrumentation-tests/build.gradle
+++ b/utbot-instrumentation-tests/build.gradle
@@ -33,6 +33,6 @@ dependencies {
 processResources {
     // We will extract this jar in `InstrumentedProcess` class.
     from(configurations.fetchInstrumentationJar) {
-        into "instrumentation-lib"
+        into "lib"
     }
 }

--- a/utbot-instrumentation/src/main/kotlin/org/utbot/instrumentation/rd/InstrumentedProcess.kt
+++ b/utbot-instrumentation/src/main/kotlin/org/utbot/instrumentation/rd/InstrumentedProcess.kt
@@ -2,19 +2,14 @@ package org.utbot.instrumentation.rd
 
 import com.jetbrains.rd.util.lifetime.Lifetime
 import mu.KotlinLogging
-import org.utbot.common.currentProcessPid
+import org.utbot.common.JarUtils
 import org.utbot.common.debug
-import org.utbot.common.firstOrNullResourceIS
 import org.utbot.common.getPid
 import org.utbot.common.measureTime
-import org.utbot.common.nameOfPackage
-import org.utbot.common.scanForResourcesContaining
-import org.utbot.common.utBotTempDirectory
 import org.utbot.framework.UtSettings
 import org.utbot.framework.plugin.api.UtModel
 import org.utbot.framework.plugin.services.WorkingDirService
 import org.utbot.framework.process.AbstractRDProcessCompanion
-import org.utbot.instrumentation.agent.DynamicClassTransformer
 import org.utbot.instrumentation.instrumentation.Instrumentation
 import org.utbot.instrumentation.process.DISABLE_SANDBOX_OPTION
 import org.utbot.instrumentation.process.generated.AddPathsParams
@@ -27,7 +22,6 @@ import org.utbot.rd.ProcessWithRdServer
 import org.utbot.rd.exceptions.InstantProcessDeathException
 import org.utbot.rd.generated.LoggerModel
 import org.utbot.rd.generated.loggerModel
-import org.utbot.rd.loggers.UtRdKLogger
 import org.utbot.rd.loggers.setup
 import org.utbot.rd.onSchedulerBlocking
 import org.utbot.rd.startBlocking
@@ -37,43 +31,25 @@ import java.io.File
 
 private val logger = KotlinLogging.logger { }
 
-private const val UTBOT_INSTRUMENTATION = "utbot-instrumentation-shadow"
-private const val INSTRUMENTATION_LIB = "lib"
-
-private fun tryFindInstrumentationJarInResources(): File? {
-    logger.debug("Trying to find jar in the resources.")
-    val tempDir = utBotTempDirectory.toFile()
-    val unzippedJarName = "$UTBOT_INSTRUMENTATION-${currentProcessPid}.jar"
-    val instrumentationJarFile = File(tempDir, unzippedJarName)
-
-    InstrumentedProcess::class.java.classLoader
-        .firstOrNullResourceIS(INSTRUMENTATION_LIB) { resourcePath ->
-            resourcePath.contains(UTBOT_INSTRUMENTATION) && resourcePath.endsWith(".jar")
-        }
-        ?.use { input ->
-            instrumentationJarFile.writeBytes(input.readBytes())
-        } ?: return null
-    return instrumentationJarFile
-}
-
-private fun tryFindInstrumentationJarOnClasspath(): File? {
-    logger.debug("Trying to find it in the classpath.")
-    return InstrumentedProcess::class.java.classLoader
-        .scanForResourcesContaining(DynamicClassTransformer::class.java.nameOfPackage)
-        .firstOrNull {
-            it.absolutePath.contains(UTBOT_INSTRUMENTATION) && it.extension == "jar"
-        }
-}
+private const val UTBOT_INSTRUMENTATION_JAR_FILENAME = "utbot-instrumentation-shadow.jar"
 
 private val instrumentationJarFile: File =
-    logger.debug().measureTime({ "Finding $UTBOT_INSTRUMENTATION jar" } ) {
-        tryFindInstrumentationJarInResources() ?: run {
-            logger.debug("Failed to find jar in the resources.")
-            tryFindInstrumentationJarOnClasspath()
-        } ?: error("""
-                    Can't find file: $UTBOT_INSTRUMENTATION.jar.
-                    Make sure you added $UTBOT_INSTRUMENTATION.jar to the resources folder from gradle.
-                """.trimIndent())
+    logger.debug().measureTime({ "Finding $UTBOT_INSTRUMENTATION_JAR_FILENAME jar" } ) {
+        try {
+            JarUtils.extractJarFileFromResources(
+                jarFileName = UTBOT_INSTRUMENTATION_JAR_FILENAME,
+                jarResourcePath = "lib/$UTBOT_INSTRUMENTATION_JAR_FILENAME",
+                targetDirectoryName = "utbot-instrumentation"
+            )
+        } catch (e: Exception) {
+            throw IllegalStateException(
+                """
+                    Can't find file: $UTBOT_INSTRUMENTATION_JAR_FILENAME.
+                    Make sure you added $UTBOT_INSTRUMENTATION_JAR_FILENAME to the resources folder from gradle.
+                """.trimIndent(),
+                e
+            )
+        }
     }
 
 class InstrumentedProcessInstantDeathException :

